src/api/app/models/event/base.rb

```diff
@@ -254,6 +254,22 @@ def watchers
       project.watched_projects.map(&:user)
```

Comment:
This needs to be adapted to get `watched_items` instead of `watched_projects` when the feature flag is enabled.

Reply:
@saraycp its fixed, good catch!

Reply:
I will add a spec to validate its working

Reply:
@saraycp okay added the test case :)